tests/modules/bases2fastq/test.yml

```
@@ -0,0 +1,18 @@
+- name: "bases2fastq"
+  command: nextflow run ./tests/modules/bases2fastq -entry test_bases2fastq -c ./tests/config/nextflow.config -c ./tests/modules/bases2fastq/nextflow.config
+  tags:
+    - "bases2fastq"
+  files:
+    - path: output/bases2fastq/output/Bases2Fastq-Sim_QC.html
+    - path: output/bases2fastq/output/Metrics.csv
+      md5sum: 0ec6da2b82e191098283474356024abf
+    - path: output/bases2fastq/output/RunManifest.json
+      md5sum: a07dce8ee25c2a6f9355b677c26b53e2
+    - path: output/bases2fastq/output/RunStats.json
+    - path: output/bases2fastq/output/UnassignedSequences.csv
+      md5sum: 11c1693830ce941b8cfb8d2431a59097
```

```suggestion
  - path: output/bases2fastq/output/UnassignedSequences.csv
    md5sum: 11c1693830ce941b8cfb8d2431a59097
```
Tests seem to pass for me too, but when I regenerate the `test.yml` I'm getting some variability in the checksums

---

It doesn't seem to change for me.